Geometry/CSCGeometry/test/testCSCGeometry_cfg.py

```diff
@@ -8,7 +8,7 @@
 # ====================
 process.load("Geometry.MuonNumbering.muonGeometryConstants_cff")
 process.load("Geometry.MuonNumbering.muonNumberingInitialization_cfi")
-process.load("Geometry.MuonCommonData.muonEndcapIdealGeometryXML_cfi")
+process.load('Configuration.Geometry.GeometryExtended_cff')
```

Comment:
@cvuosalo - it would be nice to convert the rest to use single quotes as well. It's also possible that the above files are not needed since you load a full scenario.